static/css/layout/index.less

```diff
@@ -11,6 +11,11 @@ div#contentBody,
 div.contentBody {
   padding: 0 20px 20px;
 }
+div#contentBody.type--page {
+  img {
+    max-width: 100%;
```

Comment:
This seems like a sensible default for images in any page. Is there a reason not to do so?

Reply:
Should there also be a height auto?

Reply:
If this is made generic it should be added to base or layout depending on the complexity of the selector.

Reply:
Complexity of the selector meaning?

Reply:
If selector is```img {}``` add to base css
If selector is something like ```.existing-parent-element img``` it probably belongs in layout.

Reply:
Okay thanks